### DISCUSSION TITLE
Reporting Scam Involving PolygonScan API and Fraudulent Website

### DISCUSSION THREAD

**Discussion by zwanski2019** [community/community#129122] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I am writing to report a fraudulent activity that I have encountered involving PolygonScan. Below are the details of the scam:

Scam Website: https://rr7799.net/
Fraudulent Wallet Address: 0x6662A4FfF72CAa9039045c62A78cfFeC4bDC81d4
The scammers trick victims into downloading Trust Wallet and connecting it to their website. Once the setup is complete, they connect to the victim's wallet and steal all the funds. They are stealing millions every day.
MATIC Balance: 223792792005129774 MATIC (~$5.34)
Last Transaction Sent 0x5398d7f20720e7e5fe08892cac36006e033a8c73f58d475e1d33cf44cca58535 (8 hours ago)
First Transaction Sent 0x416e233f571b18f8f3a8f0de8c36cf5b8939fec003840787d1d0f901d2847eba (76 days ago)
Evidence:find the attachments

Please investigate this matter urgently to prevent further fraudulent activities and to protect other users. If you need any additional information, please do not hesitate to contact me.

Thank you for your prompt attention to this matter.

Best regards,

zwanski
![proff](https://github.com/community/community/assets/53589819/9354d917-7165-4f25-805f-dcfe598dd769)
![proff2](https://github.com/community/community/assets/53589819/fb45465e-2d6e-46ec-8933-c4bdbcf1fe37)
![proff3](https://github.com/community/community/assets/53589819/e1a96e6e-baf6-4e8a-8548-09ee38901351)
![proff4](https://github.com/community/community/assets/53589819/08a85446-3bad-429b-8847-693b38b83ec3)
![proff5](https://github.com/community/community/assets/53589819/67d6813f-97e8-4156-aa25-ef89fd975d72)

**Comment by ebndev** [comment, marked as answer]
Hi @zwanski2019 ,

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on resolving these matters. GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic!
